web/src/components/TraceDrawer/TraceDrawerHeader.tsx

```diff
@@ -80,6 +80,9 @@ const TraceDrawerHeader: React.FC<IProps> = ({
         >
           Add Assertion
         </S.AddAssertionButton>
+        <span style={{marginLeft: 16}}>
```

Comment:
We could or not, create styled-components everytime we try to style an HTML/antd tag, let's think about that for the future.